test/tests/api/worker/facades/MailFacadeTest.ts

```diff
@@ -425,6 +428,12 @@ o.spec("MailFacade test", function () {
 			await assertThrows(ProgrammingError, async () => {
 				validateMimeTypesForAttachments([attach("a/b?c", "please stop.json")])
 			})
+			await assertThrows(ProgrammingError, async () => {
```

Comment:
you could use `o(() =>{}).throws(ProgrammingError)` in all of these

Reply:
Cool, I went and did that, then. Also seems none of the code was actually async, so I cleaned that up, too.

Reply:
yeah that's why I noticed it